src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6171,8 +6171,22 @@ namespace HVACVariableRefrigerantFlow {
 			AverageUnitMassFlow = ( PartLoadRatio * CompOnMassFlow ) + ( ( 1 - PartLoadRatio ) * CompOffMassFlow );
 			AverageOAMassFlow = ( PartLoadRatio * OACompOnMassFlow ) + ( ( 1 - PartLoadRatio ) * OACompOffMassFlow );
 		} else {
-			AverageUnitMassFlow = CompOnMassFlow;
-			AverageOAMassFlow = OACompOnMassFlow;
+			if ( VRF( VRFTU( VRFTUNum ).VRFSysNum ).VRFAlgorithmTypeNum == AlgorithmTypeSysCurve ) {
```

Comment:
@rongpengzhang I agree but it would conflict with another average mass flow rate setting when there is no load [here] (https://github.com/NREL/EnergyPlus/blob/111768812_VRFNoLoadOAFlow/src/EnergyPlus/HVACVariableRefrigerantFlow.cc#L8884) hence will results in diffs.  You might want to review that as well:

		if( PartLoadRatio == 0 ) {
			// only provide required OA when coil is off
			CompOnMassFlow = OACompOnMassFlow;
should this be:
                        CompOnMassFlow = OACompOffMassFlow;

because there could be a different OA flow rate when the coil is on and off. That why the defect file captured this issue.